README.md

```diff
@@ -1,2 +1,21 @@
-# Hello-Microverse
+# Description
 The aim of this project is to master all the tools(git and github) and best practices
+
+# Major languages
+HTML & CSS
+
+# prerequisites
+
+# Installation
+
+# Contributing
+Check out the issues tab
+
+# Citation
+
+# License
+MIT License
+
+# Author
+Agasha Bornittah 
+Github: https://github.com/Bornittah
```

Comment:
- [ ] Please don't forget to fill in those sections and have a title for the project at the top of your `README` file. Also kindly add the `Getting started` section with info on how to set up your project locally.